scijava/scijava-ops-engine/src/main/java/org/scijava/ops/engine/yaml/AbstractYAMLOpInfoCreator.java

```diff
@@ -16,24 +28,35 @@ public abstract class AbstractYAMLOpInfoCreator implements YAMLOpInfoCreator {
 
     @Override
     public OpInfo create(final URI identifier, final Map<String, Object> yaml) {
-        // Parse path - start after the leading slash
-        final String path = identifier.getPath().substring(1);
-        // Parse source
-        final String srcString = path.substring(0, path.indexOf('/'));
-        // Parse version
-        final String version = path.substring(path.indexOf('/') + 1);
+        // Parse source - start after the leading slash
+        final String srcString = identifier.getPath().substring(1);
+        // TODO: Parse version
+        final String version = "0.0";
```

Comment:
This needs to be solved - we should probably find and cache the version from the POM, if we can't guarantee that the version can be included in the YAML file.

We may just want to create another issue, since the best solution is probably just to enable the SciJava Javadoc annotation processor to parse the version from the POM. I don't know if that means releasing scijava-meta, or duplicating code, meaning this may be a bigger decision that shouldn't hold up this PR. cc @ctrueden

Reply:
I got versions figured out using a second argument to the annotation parser, which passes the version from the POM in the invocation of the annotation parser!